src/cloudant/design_document.py

```diff
@@ -243,3 +207,128 @@ def info(self):
         GET databasename/_design/{ddoc}/_info
         """
         raise NotImplementedError("_info not yet implemented")
+
+class CloudantDesignDocument(DesignDocument):
+    """
+    Encapsulates a Cloudant version of a
+    :class:`~cloudant.design_document.DesignDocument`.
+    A CloudantDesignDocument object is instantiated with a reference
+    to a Cloudant database and provides an API to query index management,
+    list and show functions, etc.
+
+    Note:  Currently only the query view management API exists.  Remaining
+    Cloudant design document functionality will be added later.
+
+    :param database: A ``CloudantDatabase`` instance used by the
```

Comment:
Isn't this the `CloudantDatabase` instance where the design document is saved? Might it be a little clearer to say that? This description is accurate, but it doesn't really help a caller much in understanding what database to provide.